CMakeLists.txt

```diff
@@ -174,44 +175,48 @@ find_package(ZLIB REQUIRED)
 # This provides the variables such as CMAKE_INSTALL_LIBDIR for installation paths.
 include(GNUInstallDirs)
 
-# Ensure rapidjson submodule is ready
-find_package(Git QUIET)
-if(GIT_FOUND AND EXISTS "${PROJECT_SOURCE_DIR}/.git")
-  # Update submodules as needed
-  option(GIT_SUBMODULE "Check submodules during build" ON)
-  if(GIT_SUBMODULE)
-    message(STATUS "Submodule update")
-    execute_process(COMMAND ${GIT_EXECUTABLE} submodule update --init --recursive
-                    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-                    RESULT_VARIABLE GIT_SUBMOD_RESULT)
-    if(NOT GIT_SUBMOD_RESULT EQUAL "0")
-      message(FATAL_ERROR "git submodule update --init failed with ${GIT_SUBMOD_RESULT}, please checkout submodules")
+if(RAPIDJSON_SYS_DEP)
+  find_package(RapidJSON 1.1.0 CONFIG REQUIRED)
```

Comment:
What happens if the version is not 1.1.0? Can we soften the requirement to be 1.x? (Or does RapidJSON not use SemVer?)

Reply:
Since `EXACT` isn't supplied, it means that any version compatible with this is valid, based on RapidJSONs version configuration. Based on their configuration anything equal to or greater than is compatible.

However, RapidJSON hasn't released in years so this is very unlikely to be an issue in this case.